src/ca65/pseudo.c

```diff
@@ -1730,6 +1730,18 @@ static void DoPushSeg (void)
 
 
 
+static void DoReferTo (void)
+/* Mark given symbol as referenced */
+{
+    SymEntry* Sym = ParseAnySymName (SYM_ALLOC_NEW);
+    if (Sym)
+    {
```

Comment:
Remember our slightly different style about where to put left braces.

Reply:
Fixed.